### PR TITLE
Respond to reviews on GitHub autolinks & CI outputs

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/118
Closes https://github.com/greenelab/meta-review/issues/121

### CODE REVIEW

File: content/response-to-reviewers.md [hunk 1]
```diff
@@ -167,6 +167,12 @@ is there anything special about how manubot links to and from issues in the text
 mostly just curious.
 (I could imagine automatically generated backreferences, as github does between issues when you mention one issue in another issue).
 
+We discuss Manubot's handling of GitHub links in [GH118](https://github.com/greenelab/meta-review/issues/118).
+Manubot does not treat references to GitHub URLs differently than any other URL.
+In addition, Manubot does not [autolink](https://help.github.com/en/articles/autolinked-references-and-urls) references to GitHub issues and pull requests.
+While some of this functionality would be possible with a Pandoc filter, this is a low priority development issue.
+One approach that we use currently include relevant issue links in commit messages, which does enable GitHub backreferences to some degree.
```
Comment (agitter) [inline content/response-to-reviewers.md]: Should this be `currently is to include`

File: content/response-to-reviewers.md [hunk 2]
```diff
@@ -179,6 +185,13 @@ e.g. I see a lot of utility in manubot-style approaches for Carpentries-style on
 I assume this happens but want to double check (and am at the far end of a lousy Internet connection right now).
 So this is really more of an "@ctb todo" item.
 
+We discuss this comment further in [GH121](https://github.com/greenelab/meta-review/issues/121).
```
Comment (agitter) [inline content/response-to-reviewers.md:188]: Are we leaving #121 open until other aspects are addressed?

File: content/response-to-reviewers.md [hunk 3]
```diff
@@ -179,6 +185,13 @@ e.g. I see a lot of utility in manubot-style approaches for Carpentries-style on
...
+Rendered versions of manuscripts do not currently get uploaded to pull requests.
+Instead, pull requests must be built locally to view the user-facing outputs (i.e. HTML or PDF).
+This is a [limitation](https://github.com/travis-ci/travis-ci/issues/5579) of pull request builds on Travis CI, which are unable to [upload artifacts](https://docs.travis-ci.com/user/uploading-artifacts/) due to security restriction.
```
Comment (agitter) [inline content/response-to-reviewers.md]: Perhaps `due to a security`

File: content/response-to-reviewers.md [hunk 4]
```diff
@@ -179,6 +185,13 @@ e.g. I see a lot of utility in manubot-style approaches for Carpentries-style on
...
+Given the importance of exposing rendered manuscripts from pull requests, we will closely monitor for possible solutions.
```
Comment (agitter) [inline content/response-to-reviewers.md:192]: I suggest here or in the next line we link to a new issue in `manubot/rootstock` to show that we are actively working on this as a high priority issue.  Getting this to work would make reviewing pull requests significantly easier.

**Review by agitter — CHANGES_REQUESTED** [review]

**Comment by agitter** [comment]
I saw this is WIP but left some initial comments anyway.  I may not be available much the rest of the weekend.

File: content/response-to-reviewers.md (hunk 2 above)
Comment (dhimmel) [reply content/response-to-reviewers.md:188]: I was planning to close it with this PR after opening a rootstock issue.

Opened at https://github.com/manubot/rootstock/issues/198

**Review by agitter — APPROVED** [review]

**Pull request merged**